### ISSUE TITLE
[Accessibility]A11y_AzureToolsStorageExplorer_ManageAccessPolicies_ManageAccessPolicies : AI4D: (StorageExplorer/table '') An element of the given ControlType must support the Grid pattern.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1880]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS4.1.2; #AI4D; #Win10;

**Environment Details:**
Windows Version: Windows10

**Steps to Reproduce:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6. Verify that all the elements of "Manage Access Policies" screen are accessible and MAS Compliant.
**Actual:**
(StorageExplorer/ 'Expiry time') The LocalizedControlType property must not be null.

Same issue is observed in below Scenarios:
**Application:**
Here in this Scenario Localized control type does not exist for controls(Application, Data Explorers, Transfers).

**Expected:**
(StorageExplorer/ 'Expiry time') The LocalizedControlType property must not be null. it should be provided.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx  

**User Impact:** Screenreader user unable to know the name of the check box.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730} 

**Attachment for reference**
![1880_ Accessibility A11y_AzureToolsStorageExplorer_ManageAccessPolicies_ManageAccessPolicies  AI4D](https://user-images.githubusercontent.com/55691739/65420143-0a051080-de1e-11e9-96b7-88e2a82e3a22.png)

**Comment by craxal** [comment]
Duplicate of #1879.

@Radha2019 The titles of these two bugs are different, but the descriptions of these bugs are pointing out the same issues. I'm closing this as duplicate. However, if they are different, please update this bug with the correct information.

**Comment by Suraj1200499** [comment]
@Craxal the bug 1879 and 1880 are different but once again we verified the issue#1880 on Application Version: 1.10.0, and the issue is not reproducing.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
